test/server/cards/01_SOR/events/PowerOfTheDarkSide.spec.ts

```diff
@@ -0,0 +1,49 @@
+describe('Power of the Dark Side', function() {
+    integration(function(contextRef) {
+        describe('Power of the Dark Side\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['power-of-the-dark-side'],
+                        groundArena: ['general-veers#blizzard-force-commander'],
+                        leader: { card: 'iden-versio#inferno-squad-commander', deployed: true }
+                    },
+                    player2: {
+                        groundArena: ['fleet-lieutenant'],
+                        leader: { card: 'sabine-wren#galvanized-revolutionary', deployed: true }
+                    }
+                });
+
+            });
+
+            it('forces opponent to defeat friendly unit', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.powerOfTheDarkSide);
+                expect(context.player2).toBeAbleToSelectExactly([context.fleetLieutenant, context.sabineWren]);
+
+                context.player2.clickCard(context.fleetLieutenant);
+                expect(context.fleetLieutenant).toBeInLocation('discard');
+            });
+
+            it('forces opponent to defeat a leader unit if nothing else is available', function () {
+                const { context } = contextRef;
+
+                context.player2.setActivePlayer();
+
+                // Attack with Fleet Lieutenant, choose Veers as target
```

Comment:
nit: for readability, instead of requiring player actions to set up the desired gamestate could we have a separate setupTest block that declares it